.github/scripts/external_pr_labeller.py

```diff
@@ -44,22 +48,25 @@ def init_logger():
     gh_repo = github.get_repo(full_name_or_id=repository_name)
 
     pr = gh_repo.get_pull(number=pr_number)
-    
+
     LOGGER.info(f'CONTEXT: PR #{pr_number}. USER: {pr.user.login}. ALL PR LABELS: {list(pr.get_labels())}')
 
     if not gh_repo.has_in_collaborators(pr.user.login):
         LOGGER.info(f'THE {pr.user.login} IS NOT A COLLABORATOR')
-        
+
         for label in pr.get_labels():
-            if label.name == EXTERNAL_PR_LABEL_NAME:
-                LOGGER.info(f'THE PR ALREADY HAS THE "{EXTERNAL_PR_LABEL_NAME}" LABEL')
+            if label.name in (ExternalPRLabels.ExternalPR, ExternalPRLabels.ExternalIntelPR):
+                LOGGER.info(f'THE PR ALREADY HAS THE "{label.name}" LABEL')
                 break
         else:
-            pr.add_to_labels(EXTERNAL_PR_LABEL_NAME)
-            LOGGER.info(f'THE "{EXTERNAL_PR_LABEL_NAME}" LABEL WAS ADDED TO THE PR')
+            is_intel_user = bool(pr.user.email and ('intel.com' in pr.user.email))
```

Comment:
How about `pr.user.email.endswith('@intel.com')`?

Reply:
is it possible for people outside of Intel org to setup such fake email  ?

Reply:
even if they somehow do, this check would not be executed as it is under the `has_in_collaborators` if.